src/prowjobsscraper/event.py

```diff
@@ -137,10 +141,20 @@ def index_prow_jobs(self, jobs: list[ProwJob]):
         job_events = (JobEvent.create_from_prow_job(j).dict() for j in jobs)
         self._jobs_index.index(job_events)
 
-    def scan_build_ids(self) -> set[str]:
+    def index_equinix_usages(self, usages: list[EquinixUsage]):
+        equinix_usages = (
+            EquinixUsageEvent.create_from_equinix_usage(u).dict() for u in usages
+        )
+        self._equinix_index.index(equinix_usages)
+
+    def scan_build_ids_from_jobs_index(self) -> set[str]:
         results = self._jobs_index.scan({"_source": ["job.build_id"]})
         return {r["_source"]["job"]["build_id"] for r in results}
 
+    def scan_build_ids_from_usages_index(self) -> set[str]:
+        results = self._equinix_index.scan({"_source": ["build_id"]})
```

Comment:
I think it would be nice to have `job.build_id` instead of `build_id`, that way the access to this information is the same in jobs, steps and usages index and can easily be correlated in kibana.

It would also allow to refactor this function with the one above as well .